third_party/open-policy-agent/gatekeeper/helmify/static/README.md

```diff
@@ -36,6 +36,8 @@ helm upgrade -n azure-workload-identity-system [RELEASE_NAME] azure-workload-ide
 | image.pullPolicy                   | Image pullPolicy                                                                                                                  | `IfNotPresent`                                          |
 | image.release                      | The image release tag to use                                                                                                      | Current release version: `v1.2.0`                       |
 | imagePullSecrets                   | Image pull secrets to use for retrieving images from private registries                                                           | `[]`                                                    |
+| goEnvVars                          | Go runtime environment variables to pass to the azure-wi-webhook-controller-manager's _manager_ container.                        | `See _values.yaml_`                                     |
```

Comment:
Instead of a new variable for go env vars, I would prefer just having the extra env vars with the go snippets added as example.